docs/tutorials/first-steps.rst

```diff
@@ -10,6 +10,24 @@ particulary relevant as a storage backend:
   - Sync and share data between users, with fined-grained permissions.
 
 
+A word about users with Kinto
+=============================
+
+First of all Kinto doesn't handle users managements.
+
+You do not have such thing as user creation, user removal, user
+password modifications, etc.
+
+However Kinto handle users permissions, which means users are uniquely
+identified in Kinto.
+
+If you are wondering how it is possible, you probably want to read
+the explanation about :ref:`authenticating with Kinto <authenticating>`.
+
+In this tutorial we will use the Basic Auth backend with random user
+tokens.
```

Comment:
In this tutorial we will use a Basic Authentication, which determines a user id based on the token provided in the request.

This method has many limitations but has the advantage to avoid specific setup or third-party services.